src/Mantasflowers.WebApi/Controllers/UserController.cs

```diff
@@ -0,0 +1,226 @@
+﻿using FirebaseAdmin.Auth;
+using Mantasflowers.Persistence.Authentication;
+using Mantasflowers.WebApi.Extensions;
+using Mantasflowers.WebApi.Responses;
+using Mantasflowers.WebApi.Setup.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mantasflowers.WebApi.Controllers
+{
+    [ApiController]
+    [Route("/firebase")]
+    public class UserController : ControllerBase
+    {
+        private readonly ILogger _logger = Log.ForContext<UserController>();
```

Comment:
I already mentioned this many times, but even more reason to move all the logic in controllers to some sort a service class. Don't really want to use loggers directly in controllers.